### PR TITLE
Left Nav Corrections in Careers

### PR DESCRIPTION
**Pull request by accbjt**
## Description
Left Nav Corrections in Careers

## Testing done
Manual Test

## Screenshots
screen shot 2018-10-08 at 1 12 53 pm

## Acceptance criteria
- [ ] Links should be displayed like the spreadsheet.

https://docs.google.com/spreadsheets/d/1SsA5RVkERpMF8DE4Oal7fy4dK617JkdJgMQ2zrRdJXc/edit#gid=203626042

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by accbjt** [comment]
Don't review yet. Have some updates to add.

**Comment by accbjt** [comment]
This is good for review now

File: src/applications/proxy-rewrite/redirects/otherDomainRedirects.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
   {
     "domain": "www.benefits.va.gov",
     "src": "/vocrehab/application_roadmap.asp",
-    "dest": "/careers-employment/how-to-apply/"
+    "dest": "/careers-employment/vocational-rehabilitation/how-to-apply/"
```
Comment (jbalboni) [inline src/applications/proxy-rewrite/redirects/otherDomainRedirects.json]: Is this updated in the URL mapping spreadsheet?

**Review by jbalboni — APPROVED** [review]
Looks ok to me

File: src/applications/proxy-rewrite/redirects/otherDomainRedirects.json (hunk 1 above)
Comment (accbjt) [reply src/applications/proxy-rewrite/redirects/otherDomainRedirects.json]: Yes, it looks like it's been updated.

**Pull request merged**